singer_sdk/plugin_base.py

```diff
@@ -48,7 +50,10 @@
 class PluginBase(metaclass=abc.ABCMeta):
     """Abstract base class for taps."""
 
-    name: str  # The executable name of the tap or target plugin.
+    name: str  # The executable name of the plugin. e.g. tap-csv
+    package_name: str | None = (
+        None  # The package name of the plugin. e.g. meltanolabs-tap-csv
+    )
```

Comment:
```suggestion
    #: The executable name of the plugin. e.g. tap-csv
    name: str

    #: The package name of the plugin. e.g. meltanolabs-tap-csv
    package_name: str | None = None
```

This will document the class attributes on the sphinx site.